src/plugins/kibana/index.js

```diff
@@ -1,3 +1,6 @@
+const Boom = require('boom');
+const { join } = require('path');
+const requireAllAndApply = require('./server/lib/require_all_and_apply');
```

Comment:
You're adding the ingest api routes in the server/routes/api/ingest/index.js file. Do you have any objection to using the requireAllAndApply function to add all of the routes in the routes directory?